Graphics/GraphicsEngineD3D11/src/ShaderD3D11Impl.cpp

```diff
@@ -54,12 +54,16 @@ static const ShaderVersion HLSLValidateShaderVersion(const ShaderVersion& Versio
 
 static const ShaderVersion GetD3D11ShaderModel(D3D_FEATURE_LEVEL d3dDeviceFeatureLevel, const ShaderVersion& HLSLVersion)
 {
+#ifdef _MSC_VER
```

Comment:
Can you please also add `!defined(NTDDI_WIN10_FE)` to this condition so that it is disabled in newer SDKs